mcsema/CFG/CFG.cpp

```diff
@@ -31,6 +31,8 @@
 #pragma clang diagnostic ignored "-Wshorten-64-to-32"
 #include <CFG.pb.h>
 #pragma clang diagnostic pop
+#include <google/protobuf/io/zero_copy_stream_impl.h>
```

Comment:
Why include the `_impl.h` file?

Reply:
That's the header where the IstreamInputStream is in:
https://developers.google.com/protocol-buffers/docs/reference/cpp/google.protobuf.io.zero_copy_stream_impl#IstreamInputStream
(`#include <google/protobuf/io/zero_copy_stream_impl.h>`)

Reply:
Alright so the protobuf source downloaded from github has the missing `google/protobuf/stubs/stl_util.h` header, but as far as I can tell, it is not actually installed. @alessandrogario any ideas on how to force more stuff from that stubs folder to be installed?

Reply:
You'd probably need to append it around here: https://github.com/trailofbits/cxx-common/blob/master/cmake/protobuf_project/libprotobuf/CMakeLists.txt#L115